### ISSUE TITLE
Questions about the “weight ”configuration of COCOA cls dataset different from D2SA and KINS.

### ISSUE THREAD

**Issue by mingheyuemankong** [YutingXiao/Amodal-Segmentation-Based-on-Visible-Region-Segmentation-and-Shape-Prior#19]
Hi, I think your work is great!
But about the configuration files of D2SA, KINS, COCOA cls, I have the following questions:
(1) The author used  _**WEIGHTS: detectron2://ImageNetPretrained/MSRA/R-50.pkl**_ in **Base-RCNN-FPN-D2SA.yaml** and **Base-RCNN-FPN-KINS.yaml**.  However, in **Base-RCNN-FPN-COCOA_cls.yaml** the author uses **_WEIGHTS: https://dl.fbaipublicfiles.com/detectron2/COCO-Detection/faster_rcnn_R_50_FPN_3x/137849458/model_final_280758.pkl_**. Why is this?
(2) Why is there such a big difference when the maximum number of steps is **70,000** and **48,000** when training **D2SA** and **KINS** datasets respectively, but the maximum number of steps is only **10,000** when training **COCOA cls** dataset?
(3) I tried to set **WEIGHTS** to **R-50.pkl** in **Base-RCNN-FPN-COCOA_cls.yaml** and train **10,000** steps instead of using **model_final_280758.pkl**, but I found that this makes the AP value on **COCOA cls** dataset become very bad and the loss cannot be reduced. Is this because it is not enough to train the **COCOA cls** dataset using **R-50.pkl** with the step number set to **10,000**?
(4) If I train the **COCOA cls** dataset without using **model_final_280758.pkl**, I cannot achieve the results in the author's article, right?
Looking forward to your patient reply!    @YutingXiao

**Comment by YutingXiao** [comment]
The main reason is that the annotations in the COCOA cls dataset are too noisy. If you just load the faster rcnn checkpoint pre-trained on the coco dataset and do not train on the COCOA cls dataset, I remember the mAP is about 45. And the performance will decrease after training on the COCOA cls dataset.

**Comment by mingheyuemankong** [comment]
This means that the more this data is trained, the worse the results are. I think this may be because the dataset has too many categories and too few images.

**Comment by YutingXiao** [comment]
Yes, this is also one of the reasons.

**Comment by mingheyuemankong** [comment]
Thank you very much for your patient answer.
